sami2py/utils.py

```diff
@@ -79,7 +79,7 @@ def generate_path(tag, lon, year, day, test=False):
         str_fmt1 = 'lon{lon:03d}'
         str_fmt2 = '{year:4d}_{day:03d}'
         archive_path = os.path.join(top_directory, tag,
-                                    str_fmt1.format(lon=lon),
+                                    str_fmt1.format(lon=int(lon)),
```

Comment:
Only the number used to generate the archive directory is an int. So SAMI2 will run at the float longitude right?

Reply:
that's right.  I should probably document that somewhere

Reply:
yup! I think that makes it clear